crates/bevy_ecs/README.md

```diff
@@ -96,7 +96,7 @@ The built in "parallel stage" considers dependencies between systems and (by def
 Bevy ECS should feel very natural for those familiar with Rust syntax:
 
 ```rust
-use bevy_ecs::prelude::*;
+use bevy::prelude::*;
```

Comment:
This is documentation for the `bevy_ecs` crate.
I think this should stay as before.

Reply:
Yeah I noticed that and reverted it :)